module/x/gravity/keeper/keeper.go

```diff
@@ -639,42 +639,42 @@ func (k Keeper) UnpackAttestationClaim(att *types.Attestation) (types.EthereumCl
 // address mapping will mean having to keep two of the same data around just to provide lookups.
 //
 // For the time being this will serve
-func (k Keeper) GetDelegateKeys(ctx sdk.Context) []*types.MsgSetOrchestratorAddress {
+func (k Keeper) GetDelegateKeys(ctx sdk.Context) []*types.MsgDelegateKeys {
 	store := ctx.KVStore(k.storeKey)
-	prefix := []byte(types.EthAddressKey)
-	iter := store.Iterator(prefixRange(prefix))
+	prfx := []byte{types.EthAddressKey}
+	iter := store.Iterator(prefixRange(prfx))
```

Comment:
this prefix range we can 🗡️

Reply:
What do you mean? We won't be storing there anymore?